src/main/java/urjc/isi/entidades/TituloIdiomas.java

```diff
@@ -0,0 +1,132 @@
+package urjc.isi.entidades;
+
+import java.util.Objects;
+
+import urjc.isi.exceptions.*;
+import java.util.StringTokenizer;
+
+import com.google.gson.JsonObject;
+
+//Solamente es la definición de la tabla
+//sus campos y como trabajar con ellos
+
+public class TituloIdiomas {
+
+	private int id;
+	private String idpelicula;
+	private String titulo;
+	private String idioma;
+	private int isDefault;
```

Comment:
El campo is default no lo entiendo, puesto que el que es por defecto, será el que esté ya en la tabla películas no?

Reply:
@DanielRiveraPaniagua El isDefault venía en la tabla de títulos en idiomas, y por tanto lo añadí a la entidad por si acababa usándolo. Finalmente no lo he usado, pero vaya, tampoco molestaba. Borrado en el commit f4a54843f012d22c84fbeac6b9cf7369ee03bb23